### ISSUE TITLE
se_resnet的pooling层输出为0

### ISSUE THREAD

**Issue by mafamily2496** [Tencent/ncnn#1127]
pytorch上的se_resnet，原始层为(avg_pool): AvgPool2d(kernel_size=7, stride=1, padding=0)
转成onnx再转成ncnn，param为Pooling          1300                     1 1 1298 1300 0=1 1=7 11=7 2=1 12=1 3=0 13=0 14=0 15=0 5=1
用最新的预编译包在855上跑，从这一层开始就没有结果了

**Comment by mafamily2496** [comment]
模型用的是https://github.com/Cadene/pretrained-models.pytorch这里面的se_resnet101

**Comment by nihui** [comment]
针对onnx模型转换的各种问题，推荐使用最新的pnnx工具转换到ncnn
In view of various problems in onnx model conversion, it is recommended to use the latest pnnx tool to convert your model to ncnn

```shell
pip install pnnx
pnnx model.onnx inputshape=[1,3,224,224]
```

详细参考文档
Detailed reference documentation
https://github.com/pnnx/pnnx
https://github.com/Tencent/ncnn/wiki/use-ncnn-with-pytorch-or-onnx#how-to-use-pnnx
